src/tools/update-fixtures.ts

```diff
@@ -85,6 +87,8 @@ async function overridePolymer(sourceDir: string) {
 }
 
 (async () => {
+  let errored = false;
```

Comment:
errored is a little vague, how about `exitCode`, set as 0, and then set as 1 in the catch handler

Reply:
Done.